src/others/components/TimelineSlider.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Grid, Slider, Typography } from "@mui/material";
+import React from "react";
+import moment from "moment";
+
+const dateDisplayFormat = 'DD.MM'
+const inputDateFormat = 'DD-MM-YYYY'
+export interface TimelineSliderProps {
+    // Timeline slider assums that dates are sorted and distinct,
+    // passed as a strings in format DD-MM-YYYY.
+    possibleDates: string[]
+}
+export const TimelineSlider = ({ possibleDates }: TimelineSliderProps) => {
+    const endDate = moment(possibleDates[possibleDates.length - 1], inputDateFormat)
```

Comment:
not sure about JS/TS behavior, but will it crash if an empty list is provided?

can we make it not crash?

Reply:
Refactored it to not display slider in case of no dates and one date.